sdcm/cluster.py

```diff
@@ -4812,6 +4875,36 @@ def _parse_cs_summary(lines):
             return {}
         return results
 
+    @staticmethod
+    def _parse_cs_results(lines):
```

Comment:
where is it used?

Reply:
it's not, it a revert of a commit which I've deleted it. (the revert if cause of different reason)

Reply:
I'll remove it from the revert